be/src/runtime/exec_env.cpp

```diff
@@ -211,6 +211,12 @@ Status ExecEnv::_init(const std::vector<StorePath>& store_paths) {
     }
     _query_rpc_pool = new PriorityThreadPool("query_rpc", query_rpc_threads, std::numeric_limits<uint32_t>::max());
 
+    int load_rpc_threads = config::internal_service_async_thread_num;
```

Comment:
use `ThreadPool` to replace the `PriorityThreadPool`

Reply:
It seems there's no significant difference here. `PriorityThreadPool` is easier to use, and more widely used in our code base.

Reply:
A threadpool with min threads 10 and max threads 1000 is used.

https://github.com/StarRocks/starrocks/blob/8860c4ab818e93476fed6b28fa74c2d7108ae0a0/be/src/runtime/exec_env.cpp#L215-220